pkg/services/app/commit.go

```diff
@@ -48,17 +36,17 @@ func (a *App) GetCommits(params CommitParams) ([]gitprovider.Commit, error) {
 	}
 
 	var commits []gitprovider.Commit
-	repoName := utils.UrlToRepoName(app.Spec.URL)
+	repoName := utils.UrlToRepoName(application.Spec.URL)
```

Comment:
Check out `NormalizedRepoURL` for this: https://github.com/weaveworks/weave-gitops/blob/8b4fc1352305212fae9077f6ac1938bab6799d03/pkg/gitproviders/provider.go#L541